frontend/app/src/modules/onchain/send/TradeRecipientAddress.vue

```diff
@@ -114,9 +115,10 @@ watch(usedAnyFocused, (focus) => {
 watch(debouncedSearchValue, async (value) => {
   const values = [];
   const privateBooks = await getAddressBookData(value);
-  values.push(...privateBooks);
-
-  if (value.endsWith('.eth')) {
+  if (privateBooks.length > 0) {
+    values.push(...privateBooks);
+  }
+  else if (value.endsWith('.eth')) {
```

Comment:
I missed this previously, do we have an API or something to figure out if something is a valid ENS domain? 

Limiting it to `.eth` is disallowing for example any web domain based ens address.